lms/static/js/commerce/views/receipt_view.js

```diff
@@ -19,11 +19,6 @@ var edx = edx || {};
             this.useEcommerceApi = this.ecommerceBasketId || this.ecommerceOrderNumber;
             _.bindAll(this, 'renderReceipt', 'renderError', 'getProviderData', 'renderProvider', 'getCourseData');
 
-            /* Mix non-conflicting functions from underscore.string (all but include, contains, and reverse) into
-             * the Underscore namespace.
-             */
-            _.mixin(_s.exports());
```

Comment:
Depending on what utils in `Underscore.string` you're using, the UI Toolkit's `StringUtils` may fill your needs. Just looking through this quickly seems like you're just using `_.sprintf`, which can be replaced by `StringUtils.interpolate` easily.

[Here's an example of a file I removed `sprintf`'s from a few weeks ago:](https://github.com/edx/edx-platform/pull/11963/files#diff-238000ddb0beda13ffab4d3a74fa162bL4)